portfolio/src/main/webapp/piano_im.js

```diff
@@ -0,0 +1,49 @@
+const notes = new Map();
+notes.set('C', 'white');
+notes.set('C#', 'black');
+notes.set('D', 'white');
+notes.set('D#', 'black');
+notes.set('E', 'white');
+notes.set('F', 'white');
+notes.set('F#', 'black');
+notes.set('G', 'white');
+notes.set('G#', 'black');
+notes.set('A', 'white');
+notes.set('A#', 'black');
+notes.set('B', 'white');
+notes.set('C2', 'white');
+notes.set('C#2', 'black');
+notes.set('D2', 'white');
+notes.set('D#2', 'black');
+notes.set('E2', 'white');
+
+const svgns = 'http://www.w3.org/2000/svg';
+const svg = document.getElementsByTagName('svg')[0];
+const whiteWidth = 100;
+const blackWidth = 30;
+const whiteHeight = '300';
+const blackHeight = '200';
+let x = 0;
+y = '0';
```

Comment:
Maybe I'm missing something - is this `y` defined somewhere? Otherwise define it here with `let y = 0`.
And since this is a number, can you use an integer instead of the '0' string?

Reply:
eslint fixed it to the const y = 0;